src/helm/clients/openai_responses_client.py

```diff
@@ -145,13 +145,18 @@ def do_it() -> Dict[str, Any]:
             if request.echo_prompt:
                 text_output += request.prompt
             for output in response["output"]:
-                output_type = output["type"]  # one of "message" or "reasoning" from API observation
-                is_reasoning_output = output_type == "reasoning"
+                output_type = output[
+                    "type"
+                ]  # one of "message" or "reasoning" from API observation, but can also include tool calls
 
-                if is_reasoning_output:
+                if output_type == "reasoning":
                     reasoning_output += "\n".join([raw_output["text"] for raw_output in output["summary"]])
-                else:
+                elif output_type == "web_search_call":
+                    reasoning_output += f"\nMade a web search call with action: {output.get('action', 'No action information was provided by the API')}\n"
```

Comment:
This is not reasoning output; either ignore this output, or call `hwarn()` here.